src/java.base/share/classes/sun/util/locale/LocaleMatcher.java

```diff
@@ -148,33 +156,6 @@
         return list;
     }
 
-    /**
-     * Removes the tag(s) which are falling in the basic exclusion range(s) i.e
-     * range(s) with q=0 and returns the updated collection. If the basic
-     * language ranges contains '*' as one of its non zero range then instead of
-     * returning all the tags, remove those which are matching the range with
-     * quality weight q=0.
-     */
-    private static Collection<String> removeTagsMatchingBasicZeroRange(
-            List<LanguageRange> zeroRange, Collection<String> tags) {
-        if (zeroRange.isEmpty()) {
-            tags = removeDuplicates(tags);
```

Comment:
Can `removeDuplicates()` now be removed? There seems to be no usage.

Reply:
Done.